From 13a60c631d7cf6c7e1926473d8069795c0def9b6 Mon Sep 17 00:00:00 2001
From: Chen Qi <Qi.Chen@windriver.com>
Date: Thu, 19 Jun 2014 15:09:56 +0100
Subject: [PATCH] aufs-util: add tool concept to Makefile for cross compiling
 purpose

In a cross compilation environment, c2sh, c2tmac and ver need to be created first.
Add a tools target to Makefile to allow for this.

Upstream-Status: Pending

Signed-off-by: Chen Qi <Qi.Chen@windriver.com>

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index dbbe43d..aff969e 100644
--- a/Makefile
+++ b/Makefile
@@ -152,7 +152,7 @@ aufs.5: aufs.in.5 c2tmac
 	chmod a-w $@
 
 c2sh c2tmac ver: CC = ${HOSTCC}
-.INTERMEDIATE: c2sh c2tmac ver
+tools: c2sh c2tmac ver
 
 install_sbin: File = auibusy aumvdown auplink mount.aufs umount.aufs
 ifeq (${Glibc},no)
